### PR TITLE
Optimize the display of the `show encrypt rules`

### PR DESCRIPTION
**Pull request by lanchengx**
Changes proposed in this pull request:
- Use empty string instead of null

### CODE REVIEW

**Comment by RaigorJiang** [comment]
@lanchengx 
There is a CI failed.

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-distsql/shardingsphere-encrypt-distsql-handler/src/main/java/org/apache/shardingsphere/encrypt/distsql/handler/query/EncryptRuleQueryResultSet.java [hunk 1]
```diff
@@ -58,14 +58,20 @@ public void init(final ShardingSphereMetaData metaData, final SQLStatement sqlSt
...
+                    each.getCipherColumn(), nullToEmptyString(each.getCipherDataType()),
+                    nullToEmptyString(each.getPlainColumn()), nullToEmptyString(each.getPlainDataType()),
+                    nullToEmptyString(each.getAssistedQueryColumn()), nullToEmptyString(each.getAssistedQueryDataType()),
                     algorithmConfiguration.getType(), PropertiesConverter.convert(algorithmConfiguration.getProps()),
                     Objects.isNull(tableRuleConfiguration.getQueryWithCipherColumn()) ? Boolean.TRUE.toString() : tableRuleConfiguration.getQueryWithCipherColumn().toString()));
         });
         return result;
     }
     
+    private <E> Object nullToEmptyString(final E obj) {
```
Comment (TeslaCN) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-distsql/shardingsphere-encrypt-distsql-handler/src/main/java/org/apache/shardingsphere/encrypt/distsql/handler/query/EncryptRuleQueryResultSet.java:71]: What is the `<E>` for?

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-distsql/shardingsphere-encrypt-distsql-handler/src/main/java/org/apache/shardingsphere/encrypt/distsql/handler/query/EncryptRuleQueryResultSet.java [hunk 2]
```diff
@@ -58,14 +58,20 @@ public void init(final ShardingSphereMetaData metaData, final SQLStatement sqlSt
...
+        return obj == null ? "" : obj;
```
Comment (TeslaCN) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-distsql/shardingsphere-encrypt-distsql-handler/src/main/java/org/apache/shardingsphere/encrypt/distsql/handler/query/EncryptRuleQueryResultSet.java:72]:
```suggestion
        return null == obj ? "" : obj;
```

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-distsql/shardingsphere-encrypt-distsql-handler/src/main/java/org/apache/shardingsphere/encrypt/distsql/handler/query/EncryptRuleQueryResultSet.java (hunk 1 above)
Comment (lanchengx) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-distsql/shardingsphere-encrypt-distsql-handler/src/main/java/org/apache/shardingsphere/encrypt/distsql/handler/query/EncryptRuleQueryResultSet.java]: it is OK

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**